### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by eeeady**
Sigh. Fixes the readme paragraphs and name of the user_list variable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 ## Usage
 
 ```hcl
     module "aws_iam_user_group" {
       source         = "trussworks/iam-user-group/aws"
-      version = "1.2.0"
+      version = "1.0.2"
 
-      user_names = ["user1", "user2"]
+      user_list = ["user1", "user2"]
```
Comment (jsclarridge) [inline README.md]: Should this reference the user_list in locals?

Comment (eeeady) [reply README.md]: Not in this particular example... fixed it in the one below.

File: README.md [hunk 2]
```diff
@@ -31,13 +28,13 @@ locals {
...
-  for_each = toset(var.user_list)
+  for_each = toset(local.user_list)
   name     = each.value
 }
 
 module "aws_iam_user_group" {
   source         = "trussworks/iam-user-group/aws"
-  version = "1.2.0"
+  version = "1.0.2"
   user_names = values(aws_iam_user.user)[*].name
```
Comment (jsclarridge) [inline README.md]: Change 'user_names' -> 'user_list'?

Comment (jsclarridge) [reply README.md]: Fixed. Looks good now.

**Pull request merged**